README.md

```diff
@@ -949,34 +950,63 @@ Speaker French Event WEB3connect :
 
 ---
 
-### List of Contributions from [Encapsulate (fka KingSuper)](https://encapsulate.xyz/)
```

Comment:
now it is removing kingsuper. please just reopen a new pr like https://github.com/celestiaorg/awesome-celestia/pull/65